website/content/v1.10/talos-guides/install/boot-assets.md

```diff
@@ -97,10 +97,12 @@ talosctl upgrade --image factory.talos.dev/installer/<new_schematic_id>:{{< rele
 
 ### Example: Raspberry Pi generic with Image Factory
 
-Let's assume we want to boot Talos on a Raspberry Pi with `iscsi-tools` system extension.
+Let's assume we want to boot Talos on a Raspberry Pi with the `iscsi-tools` system extension in the first example and with the `vc4` system extension for V3D/VC4 Broadcom VideoCore GPU support in the second example.
```

Comment:
I guess this is kept as is and add an advanced section/examples in the rpi specific example is nicer, so boot assets is kept generic